docs/release/2.x.rst

```diff
@@ -94,7 +94,8 @@ small name deviations, such as the ``.wkt`` attribute being available as a
 functions from the ``shapely.ops`` submodule such as ``polygonize()``) are
 also made available in a vectorized version as top-level function.
 
-A full list of functions can be found in the API docs. TODO add link
+See ``API REFERENCE``: https://shapely.readthedocs.io/ for a full list of functions.
```

Comment:
The link itself doesn't actually make much sense, since they already _are_ on that site (I suppose I forgot we don't have an actual link to the reference docs when I added that TODO link, we only have several pages under the "API REFERENCE" header in the sidebar). 

So maybe something like:

```suggestion
A full list of functions can be found in the API docs (see the pages listed under "API REFERENCE" in the left sidebar).
```